CenterForOpenScience/osf.io
# [WIP] removed all usages of mock library, replaced them with stdlib

## Purpose

Remove all usages of mock, library, use stdlib variant instead

## Changes

Changed `import mock` to `from unittest import mock`

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify
- Verify

What are the areas of risk?

Test may begin failing

## Documentation

## Side Effects


## Ticket
